operations/helm/charts/grafana-agent/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: grafana-agent
 description: 'Grafana Agent'
 type: application
-version: 0.27.2
```

Comment:
Bumping this will automatically publish a new chart version. IMHO it's not required straight away, let's leave it as-is.

Otherwise, we'll have to update the CHANGELOG headers to mirror that.